spec/routing/admin/categories_routing_spec.rb

```diff
@@ -0,0 +1,38 @@
+require "rails_helper"
```

Comment:
toto vyhod, taketo nizkourovnove testy nemaju pridanu hodnotu (asi su aj tka generovane)

Reply:
Ok, ale také isté testy tam sú aj pre iné modely.